src/theme/NavbarItem/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
But here we _do_ want to change the implementation, of the `NavbarItem` itself. The versions that appear in the navbar dropdown should vary depending on which section of the site you're visiting. 

This component checks the current path, and if it pattern matches to the `optimize` section, sends the new docs plugin ID through to the version dropdown. Every other page falls back to the main `docs` versions.